Hunting Queries/ASimProcess/imProcess_HostExportingMailboxAndRemovingExport.yaml

```diff
@@ -16,19 +19,19 @@ tags:
   - NOBELIUM
 query: |
 
-     // Adjust the timeframe to change the window events need to occur within to alert
```

Comment:
@YaronFruchtmann : note that the team prefers to maintain uniformity around indents. I am with them on that :-). So let's try.